docs/topics/rn-known-issues.adoc

```diff
@@ -16,48 +16,19 @@ For a complete list of all known issues, see the list of link:https://issues.red
 |Component
 |Summary
 
-|link:https://issues.redhat.com/browse/WINDUPRULE-598[WINDUPRULE-598]
-|VS Code Extension
-|Quick Fix does not fix XML rules.
+|link:https://issues.redhat.com/browse/WINDUP-3019[WINDUP-3019]
+|Migration-Application
+|The bash completion feature does not work.
 
-|link:https://issues.redhat.com/browse/WINDUPRULE-589[WINDUPRULE-589]
-|VS Code Extension
-|Right-clicking a VS Code configuration and selecting *Open* does not always open the configuration.
+|link:https://issues.redhat.com/browse/WINDUP-3104[WINDUP-3104]
+|Web UI & Windup as a Service (windup-web)
+|Web console on MacOS;  After analysis, clicking a *Short reports* link results in a "Not found" page.
 
 |link:https://issues.redhat.com/browse/WINDUPRULE-516[WINDUPRULE-516]
-|VS Code Extension
-|Quick Fix does not fix multiple rules.
-
-|link:https://issues.redhat.com/browse/WINDUP-2709[WINDUP-2709]
-|Eclipse Plugin
-|Installing the Eclipse plugin 5.0.0 on CodeReady Studio 12.16 requires Maven.
-
-|link:https://issues.redhat.com/browse/WINDUP-2692[WINDUP-2692]
-|Web UI & Windup-as-a-Service
-|When RHAMT 4.3.1 is deployed on OpenShift Container Platform with the `deploy.sh` script, an invalid route is created in the namespace.
-
-|link:https://issues.redhat.com/browse/WINDUP-2645[WINDUP-2645]
-|Eclipse Che Plugin
-|Quick Fix does not fix multiple rules.
-
-|link:https://issues.redhat.com/browse/WINDUP-2632[WINDUP-2632]
-|VS Code Extension
-|`--sourceMode` must be enabled by default.
-
-|link:https://issues.redhat.com/browse/WINDUP-2626[WINDUP-2626]
-|Eclipse Che Plugin
-|MTA Explorer commands do not work in the Eclipse Che environment.
-
-|link:https://issues.redhat.com/browse/WINDUP-2622[WINDUP-2622]
-|Web UI & Windup-as-a-Service
-|When RHAMT 4.3.1 is deployed on OpenShift Container Platform, custom rules cannot be uploaded.
-
-|link:https://issues.redhat.com/browse/WINDUP-2551[WINDUP-2551]
-|Technical Debt
-|RHAMT does not run on Zulu.
-
-|link:https://issues.redhat.com/browse/WINDUP-2521[WINDUP-2521]
-|Web UI & Windup-as-a-Service
-|When RHAMT 4.2.1 is deployed on OpenShift Container Platform, long names of HTML files for reports are truncated.
+|VS Code extension
+|Quick Fix fails to apply and show the issue when there are multiple quick fixes.
 
+|link:https://issues.redhat.com/browse/WINDUPRULE-598[WINDUPRULE-598]
+|VS Code extension
+|Quick Fix does not fix XML rules.
```

Comment:
Suggest: "Apply Quick Fix" option does not work for issues detected in XML files.